Cesium3DTilesSelection/src/RasterOverlayCollection.cpp

```diff
@@ -28,14 +41,14 @@ void RasterOverlayCollection::add(std::unique_ptr<RasterOverlay>&& pOverlay) {
   this->_overlays.push_back(std::move(pOverlay));
 
   pOverlayRaw->loadTileProvider(
-      this->_pTileset->getAsyncSystem(),
-      this->_pTileset->getExternals().pAssetAccessor,
-      this->_pTileset->getExternals().pCreditSystem,
-      this->_pTileset->getExternals().pPrepareRendererResources,
-      this->_pTileset->getExternals().pLogger);
+      this->_externals.asyncSystem,
```

Comment:
Would it be better to introduce a `RasterOverlayExternals` struct to hold these? Or perhaps just rename `TilesetExternals` to something less Tileset-specific?